astoria/common/mqtt/wrapper.py

```diff
@@ -230,12 +231,14 @@ async def wait_dependencies(self) -> None:
         if len(self._dependencies) > 0:
             LOGGER.debug("Waiting for " + ", ".join(self._dependencies))
 
-            tasks = [asyncio.gather(
-                *(event.wait() for event in self._dependency_events.values()),
-            )]
+            tasks: List[Union[asyncio.Future[Any], asyncio.Task[Any]]] = [  # type: ignore
```

Comment:
Question Why are we adding a type to then ignore it?

Reply:
Because I disagree with blindly ignoring a list's type. The mypy configuration `disallow_any_explicit = True` means we cannot actually specify this type (`asyncio.gather` returns `Future[List[Any]]`).

Reply:
Specifying the type to then ignore it isn't useful though. Perhaps separating the list definition from the `asyncio.gather`. That way we can keep the strict types for `tasks`, but just ignore the poor types for `gather`